dspace-server-webapp/src/test/java/org/dspace/app/rest/RorImportMetadataSourceServiceIT.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest;
+
+import static org.dspace.app.matcher.LambdaMatcher.matches;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasSize;
+import static org.mockito.Mockito.when;
+
+import java.io.InputStream;
+import java.nio.charset.Charset;
+import java.util.Collection;
+import java.util.Optional;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.dspace.importer.external.datamodel.ImportRecord;
+import org.dspace.importer.external.liveimportclient.service.LiveImportClientImpl;
+import org.dspace.importer.external.ror.service.RorImportMetadataSourceServiceImpl;
+import org.hamcrest.Matcher;
+import org.junit.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class RorImportMetadataSourceServiceIT extends AbstractLiveImportIntegrationTest {
+
+    @Autowired
+    private LiveImportClientImpl liveImportClient;
+
+    @Autowired
+    private RorImportMetadataSourceServiceImpl rorServiceImpl;
+
+    @Test
+    public void tesGetRecords() throws Exception {
+        context.turnOffAuthorisationSystem();
+        CloseableHttpClient originalHttpClient = liveImportClient.getHttpClient();
+        CloseableHttpClient httpClient = Mockito.mock(CloseableHttpClient.class);
+
+        try (InputStream file = getClass().getResourceAsStream("ror-records.json")) {
+
+            String jsonResponse = IOUtils.toString(file, Charset.defaultCharset());
+
+            liveImportClient.setHttpClient(httpClient);
+            CloseableHttpResponse response = mockResponse(jsonResponse, 200, "OK");
+            when(httpClient.execute(ArgumentMatchers.any())).thenReturn(response);
+
+            context.restoreAuthSystemState();
+            Collection<ImportRecord> recordsImported = rorServiceImpl.getRecords("test query", 0, 2);
+            assertThat(recordsImported, hasSize(20));
+
+            ImportRecord record = recordsImported.iterator().next();
+
+            assertThat(record.getValueList(), hasSize(11));
+
+            assertThat(
+                record.getSingleValue("organization.legalName"),
+                is("The University of Texas")
+            );
+            assertThat(record.getSingleValue("organization.identifier.ror"), is("https://ror.org/02f6dcw23"));
+            assertThat(record.getSingleValue("oairecerif.acronym"), is("UTHSCSA"));
+            assertThat(record.getSingleValue("oairecerif.identifier.url"), is("http://www.uthscsa.edu/"));
```

Comment:
The "oairecerif" field checks here can be removed. These fields shouldn't be added to DSpace.